server/scripts/2021TaxSeasonDataScript.php

```diff
@@ -0,0 +1,420 @@
+
+		// Saturdays 1-4
+		$saturdayDates = getWeeklyDatesFromRange('2021-02-20', '2021-04-03');
+		foreach ($saturdayDates as $date) {
+		
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Loren Eisely data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertBennettMartinLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Bennett Martin data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 7; // Manually obtained from PROD DB
+
+		// Sundays 1-4
+		$sundayDates = getWeeklyDatesFromRange('2021-02-07', '2021-03-14');
+		foreach ($sundayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Bennett Martin Library data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertFStreetCommunityCenterData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The F Street Community Center data has already been inserted');
+	}
+	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 8; // Manually obtained from PROD DB
+
+		// Tuesdays Feb 2 - Apr 6, 5-7
+		$tuesdayDates = getWeeklyDatesFromRange('2021-02-02', '2021-04-06');
+		foreach ($tuesdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting F Street Community Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertSoutheastCommunityCollegeData() {	
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Southeast Community College data has already been inserted');
+	}
+ 	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 10; // Manually obtained from PROD DB
+
+		// Mondays, Feb 1 - Mar 1
+		$mondayDates = getWeeklyDatesFromRange('2021-02-01', '2021-03-01');
+		foreach ($mondayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 4.5, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 4.5, 6, $siteId);
+		}
+
+		// Thursdays, Feb 4 - mar 25 or apr 8
+		$thursdayDates = getWeeklyDatesFromRange('2021-02-04', '2021-04-08');
```

Comment:
Won't this also add one on April 1? What does - mar 25 or apr 8 mean? Do we still need to verify?

Reply:
There was a discrepancy in the parameter file and the flier on the end date. It's now verified as April 8th, so April 1st should be added. I'll revise that comment